controllers/upgrade_handlers.go

```diff
@@ -62,7 +158,7 @@ func (r *ImageBasedUpgradeReconciler) handleUpgrade(ctx context.Context, ibu *lc
 	// 	 r.Log.Error(err, "Failed to apply extra manifests")
 	//	 return ctrl.Result{}, err
 	// }
-
+	r.Log.Info("Done handleUpgrade")
```

Comment:
The IBU status condition for UpgradeCompleted and UpgradeInProgress should be updated to False which indicates the upgrade fails.

Reply:
Updated